### ISSUE TITLE
Error for kinogo.by

### ISSUE THREAD

**Issue by darcosion** [soxoj/maigret#66]
Hello, 

I have the following error : 
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/aiohttp-3.7.4-py3.6-linux-x86_64.egg/aiohttp/connector.py", line 999, in _create_direct_connection
    hosts = await asyncio.shield(host_resolved)
  File "/usr/local/lib/python3.6/dist-packages/aiohttp-3.7.4-py3.6-linux-x86_64.egg/aiohttp/connector.py", line 865, in _resolve_host
    addrs = await self._resolver.resolve(host, port, family=self._family)
  File "/usr/local/lib/python3.6/dist-packages/aiohttp-3.7.4-py3.6-linux-x86_64.egg/aiohttp/resolver.py", line 36, in resolve
    flags=socket.AI_ADDRCONFIG,
  File "/usr/lib/python3.6/concurrent/futures/thread.py", line 56, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/lib/python3.6/socket.py", line 745, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -2] Name or service not known

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/maigret-0.1.14-py3.6.egg/maigret/checking.py", line 49, in get_response
    response = await request_future
  File "/usr/local/lib/python3.6/dist-packages/aiohttp-3.7.4-py3.6-linux-x86_64.egg/aiohttp/client.py", line 521, in _request
    req, traces=traces, timeout=real_timeout
  File "/usr/local/lib/python3.6/dist-packages/aiohttp-3.7.4-py3.6-linux-x86_64.egg/aiohttp/connector.py", line 535, in connect
    proto = await self._create_connection(req, traces, timeout)
  File "/usr/local/lib/python3.6/dist-packages/aiohttp-3.7.4-py3.6-linux-x86_64.egg/aiohttp/connector.py", line 892, in _create_connection
    _, proto = await self._create_direct_connection(req, traces, timeout)
  File "/usr/local/lib/python3.6/dist-packages/aiohttp-3.7.4-py3.6-linux-x86_64.egg/aiohttp/connector.py", line 1011, in _create_direct_connection
    raise ClientConnectorError(req.connection_key, exc) from exc
aiohttp.client_exceptions.ClientConnectorError: Cannot connect to host kinogo.by:443 ssl:default [Name or service not known]
```

I'm going to add this to error catching : https://github.com/soxoj/maigret/blob/main/maigret/checking.py#L61

**Comment by soxoj** [comment]
Hey. Thank you for feedback. Could you please also report you python version? `maigret --version` output will be enough.

I think it could be a specific Python 3.6 error, cause `aiohttp.client_exceptions.ClientConnectorError catching` [already exists](https://github.com/soxoj/maigret/blob/main/maigret/checking.py#L67).

So I suggest you to update Python version (if it possible) and check if error will present.

**Comment by darcosion** [comment]
Yeah, I wonder why there is a crash even a catch-all error exist...

For maigret version : 
```
$ maigret --version
maigret 0.1.14
Socid-extractor:  0.0.12
Aiohttp:  3.7.4
Requests:  2.25.1
Python:  3.6.9
```

I forgot an error when lauching it : 
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/aiohttp-3.7.4-py3.6-linux-x86_64.egg/aiohttp/connector.py", line 999, in _create_direct_connection
    hosts = await asyncio.shield(host_resolved)
  File "/usr/local/lib/python3.6/dist-packages/aiohttp-3.7.4-py3.6-linux-x86_64.egg/aiohttp/connector.py", line 865, in _resolve_host
    addrs = await self._resolver.resolve(host, port, family=self._family)
  File "/usr/local/lib/python3.6/dist-packages/aiohttp-3.7.4-py3.6-linux-x86_64.egg/aiohttp/resolver.py", line 36, in resolve
    flags=socket.AI_ADDRCONFIG,
  File "/usr/lib/python3.6/concurrent/futures/thread.py", line 56, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/lib/python3.6/socket.py", line 745, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -2] Name or service not known

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/maigret-0.1.14-py3.6.egg/maigret/checking.py", line 49, in get_response
    response = await request_future
  File "/usr/local/lib/python3.6/dist-packages/aiohttp-3.7.4-py3.6-linux-x86_64.egg/aiohttp/client.py", line 521, in _request
    req, traces=traces, timeout=real_timeout
  File "/usr/local/lib/python3.6/dist-packages/aiohttp-3.7.4-py3.6-linux-x86_64.egg/aiohttp/connector.py", line 535, in connect
    proto = await self._create_connection(req, traces, timeout)
  File "/usr/local/lib/python3.6/dist-packages/aiohttp-3.7.4-py3.6-linux-x86_64.egg/aiohttp/connector.py", line 892, in _create_connection
    _, proto = await self._create_direct_connection(req, traces, timeout)
  File "/usr/local/lib/python3.6/dist-packages/aiohttp-3.7.4-py3.6-linux-x86_64.egg/aiohttp/connector.py", line 1011, in _create_direct_connection
    raise ClientConnectorError(req.connection_key, exc) from exc
aiohttp.client_exceptions.ClientConnectorError: Cannot connect to host kinogo.by:443 ssl:default [Name or service not known]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/maigret", line 11, in <module>
    load_entry_point('maigret==0.1.14', 'console_scripts', 'maigret')()
  File "/usr/local/lib/python3.6/dist-packages/maigret-0.1.14-py3.6.egg/maigret/maigret.py", line 403, in run
    loop.run_until_complete(main())
  File "/usr/lib/python3.6/asyncio/base_events.py", line 484, in run_until_complete
    return future.result()
  File "/usr/local/lib/python3.6/dist-packages/maigret-0.1.14-py3.6.egg/maigret/maigret.py", line 333, in main
    max_connections=args.connections,
  File "/usr/local/lib/python3.6/dist-packages/maigret-0.1.14-py3.6.egg/maigret/checking.py", line 469, in maigret
    await f
  File "/usr/lib/python3.6/asyncio/tasks.py", line 458, in _wait_for_one
    return f.result()  # May raise f.exception().
  File "/usr/local/lib/python3.6/dist-packages/maigret-0.1.14-py3.6.egg/maigret/checking.py", line 100, in update_site_dict_from_response
    logger=logger)
  File "/usr/local/lib/python3.6/dist-packages/maigret-0.1.14-py3.6.egg/maigret/checking.py", line 68, in get_response
    except (ssl.SSLCertVerificationError, ssl.SSLError) as err:
AttributeError: module 'ssl' has no attribute 'SSLCertVerificationError'
```

I think it's a reason of my bug : a first crash arrive on socket.gaierror Error, and in the error catching, `except (ssl.SSLCertVerificationError, ssl.SSLError) as err:` crash also because of ssl version.

**Comment by soxoj** [comment]
Oh, I see, error caused by catching of nonexistent (in python 3.6) exception. I'll fix it!

**Comment by soxoj** [comment]
Fixed in #68 and merged in master, please, check.

**Comment by darcosion** [comment]
I've checked, it's working ! Great work 👍
